Makefile

```
@@ -29,37 +30,48 @@ setup-dev: ## Setup development environment
 
 check: ## Lint and format code
 	@echo "Linting API..."
-	@poetry run ruff check src/gentropy .
+	@uv run ruff check src/gentropy .
 	@echo "Linting docstrings..."
-	@poetry run pydoclint --config=pyproject.toml src
-	@poetry run pydoclint --config=pyproject.toml --skip-checking-short-docstrings=true tests
+	@uv run pydoclint --config=pyproject.toml src
+	@uv run pydoclint --config=pyproject.toml --skip-checking-short-docstrings=true tests
 
 test: ## Run tests
 	@echo "Running Tests..."
-	@poetry run pytest
+	@uv run pytest
```

Please test this yourself, but the current command goes idle without running any tests
```suggestion
	@uv run pytest .
```

---

The tests still go ok on my side, although it seems that the long lookup, not sure why the dot helps here, assuming that we already add the `testpath(s)` to the pytest options, the dot just overwrites it. While testing your solution I got following errors cache related:
``` 
import file mismatch:
imported module 'a_creating_spark_session' has this __file__ attribute:
  /home/mindos/Projects/OpenTargets/gentropy/docs/src_snippets/howto/python_api/a_creating_spark_session.py
which is not the same as the test file we want to collect:
  /home/mindos/Projects/OpenTargets/gentropy/site/src_snippets/howto/python_api/a_creating_spark_session.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```

This seems to be due to the fact that I have previously generated the docs that contain some test duplicate.

On the note of test collection speeds:

```
(base)  mindos@mindos  ~/Projects/OpenTargets/gentropy   pyspark-bump ±  time uv run pytest --collect-only . 1> /dev/null
uv run pytest --collect-only . > /dev/null  11,81s user 2,56s system 115% cpu 12,390 total
(base)  mindos@mindos  ~/Projects/OpenTargets/gentropy   pyspark-bump ±  time uv run pytest --collect-only 1> /dev/null 
uv run pytest --collect-only > /dev/null  11,70s user 2,62s system 116% cpu 12,269 total
```